webpack.config.ts

```diff
@@ -58,7 +59,12 @@ const config: Configuration = {
   },
   devServer: {
     static: './dist',
-    port: 9001,
+    port: 9443,
+    https: true,
+    // {
+    //  key: fs.readFileSync('/var/serving-cert/plugin-serving-cert/key.pem'),
+    //  cert: fs.readFileSync('/var/serving-cert/plugin-serving-cert/cert.pem'),
+    //},
```

Comment:
Should these be removed?

Reply:
yes! i will update the PR by removing the comment lines.